src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,13 @@
+.catalog {
+  width: 944px;
```

Comment:
it's not good to hardcode width for the catalog, it's better to use a formula to calculate it like this: 
`max-width: ($card-width*$card-per-row)
    +($hor-padding*2)+($gap*$card-per-row)-$gap;`